client/src/components/DfSelectField.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Box, FormField, Select } from "grommet";
+
+const DfSelectField = (props) => {
+  const {
+    a11yTitle,
+    name,
+    id,
+    disabled,
+    help,
+    info,
+    children,
+    clear,
+    size,
+    multiple,
+    options,
+    placeholder,
+    label,
+    defaultValue,
+    required,
+    error,
+    onChange: onChangeValue,
+    ...rest
+  } = props;
+  const errorMessage = error;
+  return (
+    <Box direction="column" fill="horizontal">
+      <FormField
+        label={label}
+        htmlFor={id || `DfSelectField_${name}`}
+        error={errorMessage}
+        help={help}
+        info={info}
+        required={required}
+        disabled={disabled}
+        fill="horizontal"
+        margin={label ? undefined : { top: "xsmall" }}
+      >
+        <Select
+          name={name}
+          data-testid={id || `DfSelectField_${name}`}
+          a11yTitle={a11yTitle}
+          placeholder={placeholder}
+          options={["India", "USA", "Ukraine"]}
```

Comment:
We need to support all countries. You can get the list from my other PR here https://github.com/hpe-hcss/dfaas-app/commit/5ca67cff75f5702ae7e3a36e5d7aae99c26151a1
Also don't you need to send the value only, not the actual country name?